### PR TITLE
History table should be populated from the databses when the the Proj…

### PR DESCRIPTION
**Pull request by UzairURK**
…ect is run #5
Close issue #5 
set the path of reading DB.CSV accordingly

### CODE REVIEW

File: LinkSqueezer/src/GUI/MainFrame.java [hunk 1]
```diff
@@ -326,6 +333,26 @@ public void run() {
             }
         });
     }
+    
+    // Read and Display History
+private void populateTableFromCSV() {
+    String csvFile = "D:\\Semester #6\\Software Construction\\SemesterProject\\linkSqueezer\\LinkSqueezer\\DB.csv";
```
Comment (hussainashiqktk) [inline LinkSqueezer/src/GUI/MainFrame.java]: do not hardcode this. Give it a relative path. The DB file is the root of the project folder. Givet its relative path. Please.

**Review by hussainashiqktk — CHANGES_REQUESTED** [review]
Change the path of the Db.csv to RELATIVE.

File: LinkSqueezer/src/GUI/MainFrame.java (hunk 1 above)
Comment (UzairURK) [reply LinkSqueezer/src/GUI/MainFrame.java]: set relative path succesfully.

Screenshot (81)

**Comment by hussainashiqktk** [comment]
image

The lias and squeezedURL in wrong cloums

**Pull request merged**